app/scripts/ui.js

```diff
@@ -245,6 +245,13 @@ async function start() {
         window.document?.documentElement?.classList.add('controller-loaded');
       }
 
+      // Add custom measurement to Sentry
+      Sentry.setMeasurement(
```

Comment:
Do we need this?

My understanding is this will add a property to every automated trace however we already have a dedicated `UI Startup` custom trace with specific nested spans for the durations we want to track.

Meaning we'll also generate those `UI Startup` custom traces from each E2E test, but that will include alternate spans for things like loading scripts and first render.

Reply:
@davidmurdoch this was your request to put a `ui.initialized` measurement here.  Is it no longer necessary due to other changes?

Reply:
Yeah, it looks like it's about the same measurement as what landed in https://github.com/MetaMask/metamask-extension/pull/26636